tools/SourceKit/cmake/modules/AddSwiftSourceKit.cmake

```diff
@@ -158,6 +158,8 @@ function(add_sourcekit_swift_runtime_link_flags target path HAS_SWIFT_MODULES)
   if(SWIFT_SWIFT_PARSER)
```

Comment:
@rxwei / @DougGregor is there anything else we should be checking here, or are all the compiler plugin checks really based on `SWIFT_SWIFT_PARSER`? `lib/CompilerPluginSupport/CMakeLists.txt` seems to be using it so I assume that is the case, just double checking.

Reply:
`SWIFT_SWIFT_PARSER` implies that ASTGen is available, which compiler plugins used to depend on (for calling the `allMacros` property). I don't think that's needed anymore for compiler plugins because we got rid of the `allMacros` property.

Reply:
So nothing in compiler plugins depends on `SwiftParser` now? In that case, what should be used as the condition here? Or should we just always be adding `${SWIFTLIB_DIR}/host`?